### PR TITLE
python3Packages.papis: provide as a library too

### PR DESCRIPTION
**Pull request by teto**
The project is turning into a dependency for several scripts and possibly UIs
(see the different repositories at https://github.com/papis/) so it makes sense
to have it as a library.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Ma27** [comment]
Shouldn't `papis` be built with `buildPythonPackage` then?

**Comment by teto** [comment]
fixed it thanks

**Review by FRidh — COMMENTED** [review]
Expressions of libraries belong under `python-modules/`

**Comment by teto** [comment]
I was hesitant to move the files but I did it.

File: pkgs/development/python-modules/papis/default.nix [hunk 1]
```diff
@@ -2,7 +2,7 @@
 , python3, xdg_utils
```
Comment (dotlambda) [inline pkgs/development/python-modules/papis/default.nix]:
```suggestion
, xdg_utils
```

File: pkgs/development/python-modules/papis/default.nix [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 }:
 
-python3.pkgs.buildPythonApplication rec {
+python3.pkgs.buildPythonPackage rec {
```
Comment (dotlambda) [inline pkgs/development/python-modules/papis/default.nix]:
```suggestion
buildPythonPackage rec {
```

**Review by dotlambda — CHANGES_REQUESTED** [review]
You need to add each Python dependency as an argument and you should specify `disabled = !isPy3k`.

**Comment by teto** [comment]
I've taken the review into account and added myself as maintainer. I have a good knowledge of the upcoming papis changes so I should do the next nix update.

**Pull request merged**